src/channels/model/ChannelsQueryRequestByBusinessId.ts

```diff
@@ -0,0 +1,5 @@
+import PaginationQueryRequest from '../../model/PaginationQueryRequest';
+
+export default interface ChannelsQueryRequestByBusinessId extends PaginationQueryRequest {
```

Comment:
At the moment, retrieving channels for a business does not support pagination. `businessId` is the only accepted query parameter.

Please update this interface so it does not extend PaginationQueryRequest.